packages/agent-infra/browser/src/types.ts

```diff
@@ -130,6 +130,14 @@ export interface BrowserInterface {
    */
   launch(options?: LaunchOptions): Promise<void>;
 
+  /**
+   * Connect to an existing browser instance via WebSocket endpoint
+   * @param {Object} options - Connection options
+   * @param {string} options.browserWSEndpoint - WebSocket endpoint URL
+   * @returns {Promise<void>} Promise resolving when connection is established
+   */
+  connect(options: { browserWSEndpoint: string }): Promise<void>;
```

Comment:
Support `cdpEndpoint` might be better?  @ulivz

Reply:
https://github.com/bytedance/UI-TARS-desktop/pull/603/commits/ba6836c9cc49d3f72eb34b8a2a48cf9d1b998139